pkg/proxy/logrecord.go

```diff
@@ -79,3 +78,8 @@ func (l *HTTPLogRecord) log(typ accesslog.FlowType, verdict accesslog.FlowVerdic
 
 	l.Log()
 }
+
+func (l *HTTPLogRecord) log(typ accesslog.FlowType, verdict accesslog.FlowVerdict, code int, info string) {
+	l.Timestamp = time.Now().UTC().Format(time.RFC3339Nano)
+	l.log(typ, verdict, code, info)
```

Comment:
Is this an infinite loop? I think it should be logStamped (above) but maybe there's another layer of indirection I missed.

Reply:
Oops, thanks for spotting this. Turns out I only tested with Envoy, hence this was never called. Changed to logStamped().